examples/formbot/actions.py

```diff
@@ -85,8 +90,14 @@ def validate(self,
         # extract requested slot
         slot_to_fill = tracker.get_slot(REQUESTED_SLOT)
         if slot_to_fill:
-            slot_values.update(self.extract_requested_slot(dispatcher,
-                                                           tracker, domain))
+            for slot, value in self.extract_requested_slot(dispatcher,
```

Comment:
I think if you use `validate_{slot}` helper, you don't need to override `validate` function, all the logic below should got in these helpers